[draft] feat(cli): remove cmds

joshuef opened the pull request:
BREAKING CHANGE: this removes the files upload command in order to focus uploads
via the  cmd, which manages the local account packet info etc.## Description

reviewpad:summary

davidrusu commented on sn_interface/src/network_knowledge/mod.rs:
```diff
@@ -312,6 +312,10 @@ impl NetworkKnowledge {
     }
 
+    pub fn anti_entropy_probe(&self) -> SystemMsg {
+        SystemMsg::AntiEntropyProbe(self.section_key())
```
The AE probe will grow in the future to include the membership generation, just moving it here in prep for that.

davidrusu commented on sn_node/src/node/flow_ctrl/cmd_ctrl.rs:
```diff
@@ -78,32 +76,13 @@ impl CmdCtrl {
     }
 
-    // consume self
-    // NB that clones could still exist, however they would be in the disconnected state
```
This used to be true when we were doing multi-threading, no longer true now. There is just a single `CmdCtrl` reference alive for the duration of the node process.

davidrusu commented on sn_node/src/node/flow_ctrl/periodic_checks.rs:
```diff
@@ -235,20 +226,12 @@ impl FlowCtrl {
-        } else {
-            None
-        }
+        node.generate_section_probe_msg()
```
For some reason we were not performing probes before the first split

joshuef commented:
bors r+

joshuef committed:
feat(cli): remove  cmds

BREAKING CHANGE: this removes the files upload command in order to focus uploads
via the  cmd, which manages the local account packet info etc.

joshuef committed:
feat(cli): remove download-all  default

BREAKING CHANGE: focuses download behaviours through the  cmd, which handles syncing
and data management via account packet (local or live)

joshuef commented:
lets see what ci tests break now...
